src/daemon/rpmostreed-deployment-utils.c

```diff
@@ -229,7 +247,11 @@ rpmostreed_deployment_generate_variant (OstreeSysroot *sysroot,
       variant_add_commit_details (&dict, "base-", base_commit);
     }
   else
-    base_checksum = g_strdup (csum);
+    {
+      base_checksum = g_strdup (csum);
+      /* we would not consider endoflife attribute for layered commits */
```

Comment:
Hmm, I think it still makes sense to allow pkg-layering on the end-of-life commit though. E.g. if a user isn't ready to rebase to the new branch, we don't want to prevent them from staying there as long as they need to by adding artificial limits, right? The `status` output will still be there to remind them that they're at the end of the line. :)